public/js/app.js

```diff
@@ -0,0 +1,315 @@
+                    scrollTop: pos
+                }, 1000);
+                
+                if ($('#hamburger-overlay').length) {
+                    $('#hamburger').removeClass('hamburger-activated');
+                }
+
+            });
+
+
+            /**************** Competences items ***************/
+
+            $(window).scroll(function () {
+                var wPos = $(window).scrollTop();
+                var rdPos = $('.compItems').offset().top;
+
+                if (wPos + 500 >= rdPos) {
+                    var art = $('.compItems').find('li');
+                    art.addClass('inView');
+                }
+                
+            });
+
+            /********** Formulaire contact ************/
+            $('input').blur(function () {
+                if ($(this).val() == '') {
+                    $(this).addClass('wrong');
+                } else {
+                    $(this).removeClass('worng');
+                }
+            });
+
+            $("#contactForm").submit(function (e) {
+                e.preventDefault();
+                var nom = $("#nom").val();
+                var email = $("#email").val();
+                var message = $("#message").val();
+                var dataString = nom + email + message;
+                var msg;
+
+                if (dataString == "") {
+                    msg = $('<div class="echec">Vous devez remplir tous les champs</div>');
+
+                } else if (nom == "") {
+                    msg = $('<div class="echec">Vous devez saisir vos nom et prénom</div>');
+                } else if (email == "") {
+                    msg = $('<div class="echec">Vous devez saisir votre mail</div>');
+                } else if (message == "") {
+                    msg = $('<div class="echec">Vous devez saisir un message</div>');
+                } else {
+                    $.ajax({
+                        type: "POST",
+                        url: $(this).attr("action"),
+                        data: $(this).serialize(),
+                        success: function () {
+
+                            var msgSuccess;
+                            msgSuccess = $('<div class="merci">Merci ' + nom + ', votre message a bien été envoyé </div>');
+                            $('body').append(msgSuccess);
+                            setTimeout(function () {
+                                    $(msgSuccess).fadeOut();
+                                },
+                                2300
+                            );
+
+                            document.getElementById("nom").value = "";
+                            document.getElementById("email").value = "";
+                            document.getElementById("message").value = "";
+                        },
+                        error: function () {
+                            var msgError;
+                            msgError = $('<div class="echec">Le message n\'a pas été envoyé </div>');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/vanmarcke/ocr_projet5_blog/pullRequest?prid=8116981)
```suggestion
                            msgError = $("<div class=\"echec\">Le message n'a pas été envoyé </div>");
```